src/containers/AboutUs/AboutUs.tsx

```diff
@@ -92,29 +111,32 @@ export const AboutUs: React.FC = () => {
   const { FACEBOOK_LINK, SKYPE, PHONE } = envConfigs;
 
   return (
-    <ContainerWrapperStyled>
-      <BackButton path={RoutePathConst.Login} />
-      <ContainerStyled>
-        <AboutOurCompany>
-          <H1 isBold={true}>{t('ABOUT_US_PAGE.TITLE_1')}</H1>
-          <AboutUsContainer>
-            <Description>{t('ABOUT_US_PAGE.PARAGRAPH_1')}</Description>
-            <Description>{t('ABOUT_US_PAGE.PARAGRAPH_2')}</Description>
-          </AboutUsContainer>
-          <AboutUsContainer>
-            <TitleToDescription isBold={true}>{t('ABOUT_US_PAGE.SUBTITLE_1')}</TitleToDescription>
-            <Description>{t('ABOUT_US_PAGE.PARAGRAPH_3')}</Description>
-          </AboutUsContainer>
-        </AboutOurCompany>
-        <Footer>
-          <FooterTitle isBold={true}>{t('ABOUT_US_PAGE.CONTACT_US')}</FooterTitle>
-          <ContactUs>
-            <ContactData title='ABOUT_US_PAGE.PHONE_NUMBER' data={PHONE} />
-            <ContactData title='ABOUT_US_PAGE.SKYPE' data={SKYPE} />
-            <ContactData title='ABOUT_US_PAGE.FACEBOOK' data={FACEBOOK_LINK} />
-          </ContactUs>
-        </Footer>
-      </ContainerStyled>
-    </ContainerWrapperStyled>
+    <>
+      <ContainerImageStyled />
+      <ContainerWrapperStyled>
+        <BackButton path={RoutePathConst.Login} />
+        <ContainerStyled>
+          <AboutOurCompany>
+            <H1 isBold={true}>{t('ABOUT_US_PAGE.TITLE_1')}</H1>
+            <AboutUsContainer>
+              <Description>{t('ABOUT_US_PAGE.PARAGRAPH_1')}</Description>
```

Comment:
check this please
maybe, would be better to use /n in the json translation file